stylesheet/stylesheet.css

```diff
@@ -24,6 +24,12 @@
     padding: 112.5px; 
 }
 
+#social-icons {
```

Comment:
Using fixed positions is a bad practice, as it looks differently on different monitor sizes. The best option would be to place `social-icons` div right in the middle of the screen. This could be done by specifying `margin:0 auto`(only).